package.json

```diff
@@ -277,6 +279,11 @@
       ]
     }
   },
+  "postcss": {
+    "plugins": [
+      "autoprefixer"
```

Comment:
Didn't we decide that we didn't need autoprefixer?

Reply:
Leftover. Removed